### DISCUSSION TITLE
New Features 🤯

### DISCUSSION THREAD

**Discussion by RajSolai** [RajSolai/TextSnatcher#17] [Ideas]
Hello there, I am creating this new Discussion channel for new features.
Feel free to add up newer features, requests, ideas and more :)
Thanks for the support.

**Comment by albanobattistella** [comment]
HI,

could you please add italian language? and po for translate the interface?
Thanks!
Albano

**Comment by meyappat303** [comment]
Great App. 

Would be nice for a helper app to be accessible from the Gnome System Tray to start Text Snatcher, and also hotkey access like be a nice feature,

**Reply by RajSolai** [reply]
good idea bro
